### DISCUSSION TITLE
An error has occured but cant see the error

### DISCUSSION THREAD

**Discussion by Hallacairion** [LSPosed/MagiskOnWSALocal#296] [Q&A] [answered]
Im new to git and dont get what Iam doing wrong here maybe im just stupid

$ ./build.sh
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
Build: RELEASE_TYPE=Retail
ROOT/SUDO is required to run this script
Build: an error has occurred, exit

running the whole thing in git bash on win11
started git bash with admin rights

thanks for the help

**Comment by Howard20181** [comment, marked as answer]
You can't run it in git bash, you need to run it in WSL2.

**Reply by Hallacairion** [reply]
Alright that explains it thanks for the help.
Ill try that then

**Reply by Hallacairion** [reply]
Got it to work with that runs like a charm thank you ver much again
